### PR TITLE
fix: build windows

### PR DESCRIPTION
**Pull request by thesayyn**
Fixes https://github.com/aspect-build/bsdtar-prebuilt/issues/3

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          bazel build //:for_all_platforms
+          mv bazel-out/darwin_arm64-opt/bin/external/libarchive\~/tar/bsdtar tar_darwin_arm64
```
Comment (alexeagle) [inline .github/workflows/build.yaml:13]: is there some symmetric delete of these lines?

Comment (thesayyn) [reply .github/workflows/build.yaml:13]: i don't think so, they are all different :(

Comment (alexeagle) [reply .github/workflows/build.yaml:26]: I mean, where's the method we were using before?

Comment (thesayyn) [reply .github/workflows/build.yaml:26]: are you asking how this was working before? https://github.com/aspect-build/bsdtar-prebuilt/blob/fa28210938d7c069194ec92a34643c31eaaeb9c2/.github/workflows/release.yml#L12-L19

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -4,21 +4,24 @@ on:
...
 jobs:
+  build_all:
+    uses: ./.github/workflows/build.yaml
   release:
     runs-on: ubuntu-latest
+    needs: build_all
     steps:
       - uses: actions/checkout@v3
       - run: |
-          bazel build //:for_all_platforms
```
Comment (alexeagle) [inline .github/workflows/release.yml:14]: got it thanks - you're right I got the notification on a draft PR and this bit just wasn't there yet.

**Review by alexeagle — APPROVED** [review]
Thank you! Can haz aspect-cli OSS release for Windoze next?

**Comment by alexeagle** [comment]
I'll try to green this up

File: .github/workflows/build.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  pull_request:
+  workflow_call:
+    outputs:
+      windows:
+        description: "Windows binaries"
+        value: ${{jobs.build_windows.outputs.artifact}}
+      linux:
+        description: "Linux binaries"
+        value: ${{jobs.build_windows.outputs.artifact}}
```
Comment (alexeagle) [inline .github/workflows/build.yaml:15]: I think this line was the mistake @thesayyn  :)

Comment (thesayyn) [reply .github/workflows/build.yaml]: i am not good at multitasking i guess :/

**Comment by thesayyn** [comment]
@alexeagle don't forget to make release.yaml only run on tags when you are done.

**Review by alexeagle — APPROVED** [review]

**Pull request merged**